don't log as error if multiple source and triple is present in the log file

This is not an error, but a limitation in our processing script (that linking is not handled).
Make this a debug message.

[ERROR] [18:09] - Multiple entires in compile database with the same (source, target) pair: (/home/ednikru/work/codechecker/TinyXML/xmltest.cpp, x86_64-linux-gnu)

--- a/libcodechecker/analyze/analysis_manager.py
+++ b/libcodechecker/analyze/analysis_manager.py
@@ -472,12 +472,12 @@ def create_actions_map(actions):
     result = dict()
     for act in actions:
         if act.source_count > 1:
-            LOG.error("Multiple sources for one build action: " +
+            LOG.debug("Multiple sources for one build action: " +
                       str(act.sources))
         source = os.path.join(act.directory, act.sources.next())
         key = source, act.target
         if key in result:
-            LOG.error("Multiple entires in compile database "
+            LOG.debug("Multiple entires in compile database "
                       "with the same (source, target) pair: (%s, %s)" % key)
         result[key] = act
     return result

